packages/manager/modules/vps/src/migration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,23 @@
+{
+  "vps_migration_title": "Migration VPS",
+  "vps_migration_button": "Programmer la migration",
+  "vps_migration_description1": "Votre ancien VPS va étre remplacé par un VPS équivalent de notre",
+  "vps_migration_description1_new_range":"novelle gamme",
+  "vps_migration_description2": "Le prix restera inchangé, voir il se peut que celui-ci vous coûte un peu moins.",
+  "vps_migration_description3": "Vous avez reçu une communication qui vous indique qu'il est indispensable que vous migriez votre VPS avant le ",
+  "vps_migration_description_date": "27 juillet 2020, passé cette date votre VPS sera automatiquement résilié.",
```

Comment:
Date should be from the API

Reply:
I think this is fixed date. Doesnt come from the API

Reply:
but it should match with API

Reply:
The final day is 27 not 30 returned by API. We can hard code it.